src/main/resources/application.conf

```
@@ -44,3 +44,14 @@ mariadb {
     username = ${?MARIADB_USERNAME}
     password = ${?MARIADB_PASSWORD}
 }
+
+db-dispatcher {
+    type = Dispatcher
+    executor = "fork-join-executor"
+    fork-join-executor {
+        parallelism-min = ${?DISP_PARALLELISM_MIN}
```

more explicit name would be better
```suggestion
        parallelism-min = ${?DB_DISP_PARALLELISM_MIN}
```
or
```suggestion
        parallelism-min = ${?STS_DB_DISP_PARALL_MIN}
```